lib/process-services/src/lib/task-list/components/task-header/task-header.component.ts

```diff
@@ -72,30 +89,59 @@ export class TaskHeaderComponent implements OnChanges, OnInit {
     dateLocale: string;
 
     private currentUserId: number;
+    private readonly onDestroy$ = new Subject<void>();
+    private readonly usersSubject$ = new BehaviorSubject<CardViewSelectItemOption<number>[]>([]);
+    users$ = this.usersSubject$.asObservable();
 
     constructor(
         private peopleProcessService: PeopleProcessService,
         private translationService: TranslationService,
-        private appConfig: AppConfigService
+        private readonly appConfig: AppConfigService,
+        private readonly cardViewUpdateService: CardViewUpdateService
     ) {
         this.dateFormat = this.appConfig.get('dateValues.defaultDateFormat');
         this.dateLocale = this.appConfig.get('dateValues.defaultDateLocale');
     }
 
     ngOnInit() {
-        this.loadCurrentBpmUserId();
-        this.initData();
+        this.peopleProcessService.getCurrentUserInfo().subscribe((res) => {
+            this.currentUserId = res ? +res.id : null;
+            this.initData();
+        });
+
+        this.cardViewUpdateService.autocompleteInputValue$
+            .pipe(
+                filter((res) => res.length > 0),
+                debounceTime(300),
+                switchMap((res) => this.getUsers(res)),
+                takeUntil(this.onDestroy$)
```

Comment:
Since ng16 you can use `takeUntilDestroyed`

Reply:
good to know, I'll check this one

Reply:
Please also fix this comment